vignettes/tune.Rmd

```diff
@@ -82,10 +82,10 @@ ames_rec <-
   step_ns(Latitude,  deg_free = tune("lat df"))
 ```
 
-The function `dials::parameters()` can detect and collect the parameters that have been flagged for tuning:
+The function `hardhat::extract_parameter_set_dials()` can detect and collect the parameters that have been flagged for tuning. *This method replaces the function `dials::parameters()` deprecated in tune 0.1.6.9003
```

Comment:
```suggestion
The function `extract_parameter_set_dials()` can detect and collect the parameters that have been flagged for tuning.
```

Since this function was deprecated a couple years ago, I'd say we can go ahead and just not mention it here!

Also, since tune re-exports `extract_parameter_set_dials()`, we don't need to namespace it with `hardhat::`. :)